packages/noya-state/src/reducers/application.ts

```diff
@@ -249,59 +258,147 @@ export function reducer(
         state.selectedPage = action[1];
       });
     }
-    case 'alignLeft': {
+    case 'distribute': {
       const pageIndex = getCurrentPageIndex(state);
       const layerIndexPaths = getSelectedLayerIndexPaths(state);
+      const axis = action[1];
```

Comment:
Minor: if I'm going to rename an action parameter, I usually do 

```ts
const [, axis] = action;
```

since this pattern works with any number of parameters